stable/redis/templates/deployment.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: extensions/v1beta1
```

Comment:
@viglesiasce so this is redis in non ha mode, and even no slaves. So how do we name this puppy? We need ha with slaves and ha clustered. That would give us three different charts.